ar_commands.rb

```diff
@@ -5,27 +5,30 @@
 require_relative "models/painting"
 
 # 1. Create a new Artist (Vincent Van Gogh) and save him to your database
-
+Artist.create(name: "Vincent Van Gogh");
 # 2. Create a new Painting (Starry Night) and save it to your database
-
+Painting.create(title: "Starry Night");
 # 3. Return all Artists
-
+Artist.all;
 # 4. Return all Paintings
-
+Painting.all;
 # 5. Return the artist with an id of 2
-
+Artist.find(2);
 # 6. Return the artist with a name of "Vincent Van Gogh"
-
+Artist.find_by(name: "Vincent Van Gogh");
 # 7. Return all the artists from Spain
-
+Artist.where(nationality: "Spain");
 # 8. Return the painting with an id of 1
-
+Painting.find(1);
 # 9. Return the painting with a name of "Guernica"
-
+Painting.find_by(title: "Guernica");
 # 10. Return all the paintings that "Pablo Picasso" painted
-
+Painting.where(name: "Pablo Picasso");
```

Comment:
careful here, there are no paintings with a `name` attribute whose value is `"Pablo Picasso"`. What if you found the `artist` whose `name` was `"Pablo Picasso" and found all of that artists `paintings`